src/wallet/rpcwallet.cpp

```diff
@@ -1598,7 +1597,7 @@ static UniValue listsinceblock(const JSONRPCRequest& request)
     for (const std::pair<const uint256, CWalletTx>& pairWtx : pwallet->mapWallet) {
         CWalletTx tx = pairWtx.second;
 
-        if (depth == -1 || abs(tx.GetDepthInMainChain(*locked_chain)) < depth) {
```

Comment:
In commit "Remove locked_chain from GetDepthInMainChain and its callers" (dd3cb3804126abdba8684cb965d2e1dc3766b3fd)

Appears to be a mistake during rebase: lost abs() call

Reply:
Thanks to spot it, was introducted in 436ad43, corrected rebase mistake!